lib/kubernetes-deploy/runner.rb

```diff
@@ -187,16 +187,11 @@ def server_major_version
 
     # Inspect the file referenced in the kubectl stderr
     # to make it easier for developer to understand what's going on
-    def find_bad_file_from_kubectl_output(stderr)
-      # Output example:
-      # Error from server (BadRequest): error when creating "/path/to/configmap-gqq5oh.yml20170411-33615-t0t3m":
-      match = stderr.match(%r{BadRequest.*"(?<path>\/\S+\.ya?ml\S*)"})
-      return unless match
-
-      path = match[:path]
-      if path.present? && File.file?(path)
-        File.read(path)
-      end
+    def find_bad_files_from_kubectl_output(stderr)
+      # stderr often contains one or more lines like the following, from which we can extract the file path(es):
+      # Error from server (TypeOfError): error when creating "/path/to/service-gqq5oh.yml": Service "web" is invalid:
+      matches = stderr.scan(%r{"(?<path>\/\S+\.ya?ml\S*)"})
```

Comment:
We do we need the named capture group if we don't use it? 

This seems to work ok:

```
"a/foo.yaml".scan(%r{(/\S+\.ya?ml\S*)})
=> [["/foo.yaml"]]
```

Likewise, `%r{...}` doesn't need escaping for forward slashes.